c.pod_design.md

```diff
@@ -828,6 +828,7 @@ spec:
       creationTimestamp: null
       name: time-limited-job
     spec:
+      activeDeadlineSeconds: 12
```

Comment:
Can you add a comment “#add this line”? Thanks!

Reply:
> Can you add a comment “#add this line”? Thanks!

Sure. I appended the comment.